services/evidence/feedback-api-main/src/endpoint/endpoint.go

```diff
@@ -251,6 +253,13 @@ type Highlight struct {
 	Character int `json:"character,omitempty"`
 }
 
+type Hint struct {
+  Id int `json:"id,omitempty"`
+	Explanation string `json:"explanation,omitempty"`
+	Image_link string `json:"image_link,omitempty"`
+  Image_alt_text string `json:"image_alt_text,omitempty"`
```

Comment:
Some weird indentation stuff going on in here.  Go should be enforcing tabs-only, but something seems off here for some reason.